src/library/scala/collection/immutable/Vector.scala

```diff
@@ -851,16 +895,14 @@ private[immutable] trait VectorPointer[T] {
 
     // STUFF BELOW USED BY APPEND / UPDATE
 
-    private[immutable] final def copyOf(a: Array[AnyRef]): Array[AnyRef] = {
-      val copy = new Array[AnyRef](a.length)
-      java.lang.System.arraycopy(a, 0, copy, 0, a.length)
-      copy
+    @`inline` private[immutable] final def copyOf[T <: AnyRef](a: Array[T]): Array[T] = {
```

Comment:
How about we remove this method altogether?

Reply:
happy to. I did consider, but it looked easier to review like this. Will inline